src/devices/controlBoard_nws_yarp/RPCMessagesParser.cpp

```diff
@@ -482,14 +482,13 @@ void RPCMessagesParser::handleTorqueMsg(const yarp::os::Bottle& cmd,
                 break;
             }
 
-            params.bemf = b->get(0).asFloat64();
-            params.bemf_scale = b->get(1).asFloat64();
             params.ktau = b->get(2).asFloat64();
```

Comment:
If we do not add anymore `bemf` and `bemf_scale`, then `ktau` will be the first element of the vector, not the third one. Perhaps we can do the removal of bemf in a separate PR, as it could impact also non-iCub users of YARP?